README.md

```diff
@@ -80,6 +80,9 @@ ot.httpserver.active-connectors=default-http,fixed-http,my-https
 The `default-http` connector is hard-wired to Spring Boot's default connector and is less customizable;
 the rest are created by the `otj-server` code and wired to Jetty ourselves.
 
+The `boot` connector is behaving like `default-http` connector, but takes host/port from Spring Boot's default connector.
```

Comment:
So, are we creating 2 connectors? default-http and boot? Why 2?

Also, minor grammar changes:
```suggestion
The `boot` connector is the same as the  `default-http` connector, but it takes its host and port from Spring Boot's default connector.
```